src/modules/dmq_usrloc/usrloc_sync.c

```diff
@@ -72,6 +70,7 @@ static int add_contact(str aor, ucontact_info_t* ci)
 		// Search by ruid, if possible
 		res = dmq_ul.get_urecord_by_ruid(_d, dmq_ul.get_aorhash(&aor), &ci->ruid, &r, &c);
 		if (res == 0) {
+			dmq_ul.lock_udomain(_d, &aor);
```

Comment:
This addition to line 73 doesn't seem necessary, because the lock is aquired by get_urecord_by_ruid(). Or did I miss something?